packages/app/components/Nav/LeftNavBar.tsx

```diff
@@ -18,23 +19,33 @@ const MenuItem = styled(Menu.Item)`
   }
 `;
 
-const MenuItemLink = styled.a`
+const MenuItemLink = styled(Link)`
   padding: 10px 15px;
 `;
 
+interface LeftNavBarProps {
+  courseId: number;
+  queueId: number;
+}
 
-export default function LeftNavBar() {
+export default function LeftNavBar({ courseId, queueId }: LeftNavBarProps) {
   return (
     <HorizontalMenu mode="horizontal">
       <MenuItem key="today">
-        <MenuItemLink href="/today">Today</MenuItemLink>
+        <MenuItemLink href={`/class/${courseId}/today`}>Today</MenuItemLink>
       </MenuItem>
       <MenuItem key="week">
-        <MenuItemLink href="/schedule">Schedule</MenuItemLink>
-      </MenuItem>
-      <MenuItem key="queue">
-        <MenuItemLink href="/queue">Queue</MenuItemLink>
+        <MenuItemLink href={`/class/${courseId}/schedule`}>
+          Schedule
+        </MenuItemLink>
       </MenuItem>
+      {queueId && (
```

Comment:
should we remove the queue link if there's no queue? I think it should be a thing that's like "oh you need to sign up for a queue if you click on a queue that you don't have ya feel?"

Reply:
oh so the queue link actually navigates the user to the queue page of the open queue (if one exists), but if there's no queue then i think the link serves no purpose?